galasa-ui/src/tests/header.test.tsx

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright contributors to the Galasa project
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+import HomePage from '@/app/page';
+import PageHeader from '@/components/PageHeader';
+import { act, render, screen } from '@testing-library/react';
+
+jest.mock('next/headers', () => ({
+  cookies: jest.fn(() => ({
+    get: jest.fn().mockReturnValue('')
+  }))
+}))
+
+
+afterEach(() => {
+    delete process.env.NEXT_PUBLIC_GALASA_SERVICE_NAME
+})
+
+test('renders Galasa Service header title when env NEXT_PUBLIC_GALASA_SERVICE_NAME is null or blank string', () => {
+
+    process.env.NEXT_PUBLIC_GALASA_SERVICE_NAME = "";  //mocking environment variable
+    render(<PageHeader />);
+  
+    const galasaServiceName = process.env.NEXT_PUBLIC_GALASA_SERVICE_NAME?.trim() || "Galasa Service"
```

Comment:
You can hard-code the expected values in tests instead of re-using the logic that we're testing - same goes for the tests below

Reply:
Removed a unit test as it was no longer relevant, 

```
test('renders Galasa Service header title when env NEXT_PUBLIC_GALASA_SERVICE_NAME is a string with only spaces', () => {


    render(<PageHeader galasaServiceName=''/>);

    const titleElement = screen.getByText("Galasa Service")
    expect(titleElement.textContent).toBe("Galasa Service")
    expect(titleElement).toBeInTheDocument();

});

```

Because now that we are passing the service name as a prop a blank string will default to Galasa Service